ansible/install-apps.yml

```
@@ -0,0 +1,26 @@
+---
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  tasks:
+  - stat:
+      path: "/etc/apt/sources.list.d/google-chrome.list"
+    register: google_apt_exists
+    ignore_errors: yes
+  - name: Add Google Chrome key
+    apt_key:
+      url: https://dl-ssl.google.com/linux/linux_signing_key.pub
+      state: present
+  - name: Add Google Chrome directory
+    apt_repository:
+      repo: "{{ item }}"
+      state: present
+    loop:
+    - "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main"
```

I believe you still don't need to use quotes here:
```suggestion
    - deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main
```